value/bucket.go

```diff
@@ -52,9 +55,31 @@ func (b *BucketBlueprint) String() string {
 		evictionPolicy = b.EvictionPolicy
 	}
 
+	piTRGranularity := "N/A"
```

Comment:
super nit: should this be 'pitrGranularity'?

Reply:
Done.